sympy/vector/functions.py

```diff
@@ -474,3 +474,54 @@ def _path(from_object, to_object):
         from_path.append(other_path[i])
         i -= 1
     return index, from_path
+
+def orthogonalize(vlist, orthonormal=False):
+    """
```

Comment:
@moorepants . AFAIK, python doesn't allow something like `def orthogonalize(*vlist, orthonormal=False):`. (I am getting a syntax error). It allows something of the kind `def orthogonalize(vlist, orthonormal=False):` or `def orthogonalize(*vlist, **kwargs):`. I may be mistaken howerver.